src/core/symbology-ng/qgsmarkersymbollayerv2.cpp

```diff
@@ -2010,6 +2013,11 @@ QgsSymbolLayerV2* QgsFontMarkerSymbolLayerV2::create( const QgsStringMap& props
     angle = props["angle"].toDouble();
 
   QgsFontMarkerSymbolLayerV2* m = new QgsFontMarkerSymbolLayerV2( fontFamily, chr, pointSize, color, angle );
+
+  if ( props.contains( "color_border" ) )
```

Comment:
@nyalldawson , I noticed that there seems to be a border vs outline "tension" throughout the symbology code. Have we settled on using outline for border/outline?